client/webserver/site/src/js/wallets.js

```diff
@@ -510,6 +517,15 @@ export default class WalletsPage extends BasePage {
     Doc.show(a.unlock)
   }
 
+  async downloadLogs () {
+    const search = new URLSearchParams('')
+    search.append('assetid', this.reconfigAsset + '')
```

Comment:
Is this a JS trick to make a number into a string?  I feel like it would be more expressive with `${this.reconfigAsset}` or even `String(this.reconfigAsset)` even if they aren't turbo fast.

Reply:
I'll update to the template style.